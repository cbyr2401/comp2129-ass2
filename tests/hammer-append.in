APPEND a
APPEND b
APPEND c 1 2 3 4
APPEND catalan 1 2 5 14 42 132
LIST KEYS
LIST ENTRIES
SET fibonacci 1 1 2 3 5 8 13 21 34 55 89 144
LIST KEYS
LIST ENTRIES
APPEND fibonacci 233 377 610
LIST ENTRIES
BYE