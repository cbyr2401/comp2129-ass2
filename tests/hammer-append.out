> no such key

> no such key

> no such key

> no such key

> no keys

> no entries

> ok

> fibonacci

> fibonacci [1 1 2 3 5 8 13 21 34 55 89 144]

> ok

> fibonacci [1 1 2 3 5 8 13 21 34 55 89 144 233 377 610]

> bye